cheroot/test/test_server.py

```diff
@@ -76,6 +76,49 @@ def unix_sock_file():
     os.unlink(tmp_sock_fname)
 
 
+def test_prepare_makes_server_ready():
+    """Check that prepare() makes the server ready, and stop() clears it."""
+    httpserver = HTTPServer(
+        bind_addr=(ANY_INTERFACE_IPV6, EPHEMERAL_PORT),
```

Comment:
This might be unstable in case of missing IPv6.

Reply:
you're right